### ISSUE TITLE
Response example generation

### ISSUE THREAD

**Issue by jjg-dressipi** [Apipie/apipie-rails#629]
On APIPie 0.5.10, I find an issue with the automated generation of response examples.

If I modify my `rails helper` to have
```
config.filter_run(show_in_doc: true) if ENV['APIPIE_RECORD']
```
and annotate selected controller specs with `:show_in_doc`
```
it 'has status 404', :show_in_doc do
  get :index, params: params
  expect(response).to have_http_status(:not_found)
end
```
then, as expected, running
```
APIPIE_RECORD=examples bundle exec rake spec
```
executes only those specs so annotated.  But, in spite of a reassuring `Writing examples to a file` the file `doc/apipie_examples.json` is not modified.  If i delete the file then running the specs does not create it. 

If I comment out the `filter_run` then the file is created/modified, but of course it includes "too many" examples, so I have to manually edit the file to pick out the examples to use, which makes me sad.

Any ideas?

**Comment by Marri** [comment]
No ideas, but I'm having the same problem (APIPie 0.5.13 and Rails 5.14 and Ruby 2.5.1) and uncommenting the `filter_run` line is *not* working for me.

**Comment by RomanKapitonov** [comment]
This may probably be an issued with e3eab0a33eaff8e8311eee3f478feabde7d906ae which was later wrapped with 674fe36f472e9730d9d5c8c7914ed9b4c9633fc1

**Comment by matiasarenhard** [comment]
I have the same problem with Rails 5.2.1, Ruby 2.3.1, Rspec: 3.8.0, Apipie: 0.5.13 
Has anyone managed to solve it?

**Comment by iNecas** [comment]
Hi, https://github.com/Apipie/apipie-rails/pull/640 looks promising, but I need to give it a test run before accepting, Any feedback on it from other folks would help as well

**Comment by jjg-dressipi** [comment]
I can confirm that this problem is fixed as of 0.5.14 -- many thanks!
